lock-api/src/main/java/com/palantir/lock/watch/LockWatchCacheImpl.java

```diff
@@ -34,6 +29,10 @@ public LockWatchCacheImpl(LockWatchEventCache eventCache, LockWatchValueCache va
         this.valueCache = valueCache;
     }
 
+    public static LockWatchCache noop() {
```

Comment:
Since we generally call classes `NoOp`,  that implies this should be `noOp()`

Reply:
I don't even know why I do these things when I know you will complain about it 🗡️

Reply:
🔥